### DISCUSSION TITLE
tailwindcss-rails does not support the x86-freebsd platform

### DISCUSSION THREAD

**Discussion by whitehome** [rails/tailwindcss-rails#295] [General]
Tailwindcss::Commands::UnsupportedPlatformException: tailwindcss-rails does not support the x86-freebsd platform (Tailwindcss::Commands::UnsupportedPlatformException)

Is this intentional or an error that FreeBSD is not supported by TailwindCSS-rails.  If so is there a work around?

Setting up a FreeBSD 14 Rails development server.  I created my first app with --css tailwind and tailwinds-rails gave the above error when I started my app bare bones.  Here is the complete output:

19:27:47 web.1  | started with pid 39150
19:27:47 css.1  | started with pid 39151
19:27:48 web.1  | DEBUGGER: Debugger can attach via UNIX domain socket (/tmp/ruby-debug-sock-1001/ruby-debug-mww-39150)
19:27:48 web.1  | => Booting Puma
19:27:48 web.1  | => Rails 7.1.2 application starting in development 
19:27:48 web.1  | => Run `bin/rails server --help` for more startup options
19:27:48 css.1  | bin/rails aborted!
19:27:48 css.1  | Tailwindcss::Commands::UnsupportedPlatformException: tailwindcss-rails does not support the x86-freebsd platform (Tailwindcss::Commands::UnsupportedPlatformException)
19:27:48 css.1  | Please install tailwindcss following instructions at https://tailwindcss.com/docs/installation
19:27:48 css.1  | 
19:27:48 css.1  | Tasks: TOP => tailwindcss:watch
19:27:48 css.1  | (See full trace by running task with --trace)
19:27:48 web.1  | Puma starting in single mode...
19:27:48 web.1  | * Puma version: 6.4.0 (ruby 3.2.2-p53) ("The Eagle of Durango")
19:27:48 web.1  | *  Min threads: 5
19:27:48 web.1  | *  Max threads: 5
19:27:48 web.1  | *  Environment: development
19:27:48 web.1  | *          PID: 39150
19:27:48 web.1  | * Listening on http://192.168.2.9:3000
19:27:48 web.1  | Use Ctrl-C to stop
19:27:48 css.1  | exited with code 1
19:27:48 system | sending SIGTERM to all processes
19:27:48 web.1  | - Gracefully stopping, waiting for requests to finish
19:27:48 web.1  | Exiting
19:27:48 web.1  | terminated by SIGTERM

Ruby version: ruby 3.2.2 (2023-03-30 revision e51014f9c0) [i586-freebsd14.0]
Rails version: Rails 7.1.2
gem version: 3.4.10
bundler version: Bundler version 2.4.10
gem install -V tailwindcss-rails: 
HEAD https://index.rubygems.org/
200 OK
GET https://index.rubygems.org/info/tailwindcss-rails
200 OK
GET https://index.rubygems.org/quick/Marshal.4.8/tailwindcss-rails-2.0.32.gemspec.rz
200 OK
GET https://index.rubygems.org/info/rails
200 OK
GET https://index.rubygems.org/info/railties
200 OK
GET https://index.rubygems.org/info/actionpack
200 OK
GET https://index.rubygems.org/info/activesupport
200 OK
GET https://index.rubygems.org/info/irb
200 OK
GET https://index.rubygems.org/info/method_source
200 OK
GET https://index.rubygems.org/info/rackup
200 OK
GET https://index.rubygems.org/info/rake
200 OK
GET https://index.rubygems.org/info/thor
200 OK
GET https://index.rubygems.org/info/zeitwerk
200 OK
GET https://index.rubygems.org/info/actionview
200 OK
GET https://index.rubygems.org/info/nokogiri
200 OK
GET https://index.rubygems.org/info/racc
200 OK
GET https://index.rubygems.org/info/rack
200 OK
GET https://index.rubygems.org/info/rack-session
200 OK
GET https://index.rubygems.org/info/rack-test
200 OK
GET https://index.rubygems.org/info/rails-dom-testing
200 OK
GET https://index.rubygems.org/info/rails-html-sanitizer
200 OK
GET https://index.rubygems.org/info/mini_portile2
200 OK
GET https://index.rubygems.org/info/base64
200 OK
GET https://index.rubygems.org/info/bigdecimal
200 OK
GET https://index.rubygems.org/info/concurrent-ruby
200 OK
GET https://index.rubygems.org/info/connection_pool
200 OK
GET https://index.rubygems.org/info/drb
200 OK
GET https://index.rubygems.org/info/i18n
200 OK
GET https://index.rubygems.org/info/minitest
200 OK
GET https://index.rubygems.org/info/mutex_m
200 OK
GET https://index.rubygems.org/info/tzinfo
200 OK
GET https://index.rubygems.org/info/builder
200 OK
GET https://index.rubygems.org/info/erubi
200 OK
GET https://index.rubygems.org/info/loofah
200 OK
GET https://index.rubygems.org/info/mini_portile
200 OK
GET https://index.rubygems.org/info/pkg-config
200 OK
GET https://index.rubygems.org/info/crass
200 OK
GET https://index.rubygems.org/info/webrick
200 OK
GET https://index.rubygems.org/info/ruby2_keywords
200 OK
GET https://index.rubygems.org/info/rdoc
200 OK
GET https://index.rubygems.org/info/reline
200 OK
GET https://index.rubygems.org/info/io-console
200 OK
GET https://index.rubygems.org/info/hoe
200 OK
GET https://index.rubygems.org/info/json
200 OK
GET https://index.rubygems.org/info/psych
200 OK
GET https://index.rubygems.org/info/stringio
200 OK
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/MIT-LICENSE
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/README.md
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/Rakefile
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.alternates.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.cyrillic.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.extra.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.greek.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.latin-ext.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.latin.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.symbols.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-italic.vietnamese.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.alternates.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.cyrillic.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.extra.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.greek.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.latin-ext.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.latin.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.symbols.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/fonts/Inter-roman.vietnamese.var.woff2
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/app/assets/stylesheets/inter-font.css
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/exe/tailwindcss
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/controller/controller_generator.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/controller/templates/view.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/mailer/mailer_generator.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/mailer/templates/view.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/mailer/templates/view.text.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/scaffold_generator.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/templates/_form.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/templates/edit.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/templates/index.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/templates/new.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/templates/partial.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/generators/tailwindcss/scaffold/templates/show.html.erb.tt
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/install/Procfile.dev
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/install/application.tailwind.css
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/install/dev
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/install/tailwind.config.js
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/install/tailwindcss.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tailwindcss-rails.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tailwindcss/commands.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tailwindcss/engine.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tailwindcss/upstream.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tailwindcss/version.rb
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tasks/build.rake
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tasks/clobber.rake
/home/mww/.rvm/gems/ruby-3.2.2/gems/tailwindcss-rails-2.0.32/lib/tasks/install.rake
/home/mww/.rvm/gems/ruby-3.2.2/bin/tailwindcss
Successfully installed tailwindcss-rails-2.0.32
1 gem installed

gem env:

RubyGems Environment:
  - RUBYGEMS VERSION: 3.4.10
  - RUBY VERSION: 3.2.2 (2023-03-30 patchlevel 53) [i586-freebsd14.0]
  - INSTALLATION DIRECTORY: /home/mww/.rvm/gems/ruby-3.2.2
  - USER INSTALLATION DIRECTORY: /home/mww/.local/share/gem/ruby/3.2.0
  - RUBY EXECUTABLE: /home/mww/.rvm/rubies/ruby-3.2.2/bin/ruby
  - GIT EXECUTABLE: /usr/local/bin/git
  - EXECUTABLE DIRECTORY: /home/mww/.rvm/gems/ruby-3.2.2/bin
  - SPEC CACHE DIRECTORY: /home/mww/.local/share/gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /home/mww/.rvm/rubies/ruby-3.2.2/etc
  - RUBYGEMS PLATFORMS:
     - ruby
     - x86-freebsd-14
  - GEM PATHS:
     - /home/mww/.rvm/gems/ruby-3.2.2
     - /home/mww/.rvm/rubies/ruby-3.2.2/lib/ruby/gems/3.2.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => true
     - :bulk_threshold => 1000
     - "gem" => "--no-document"
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /home/mww/.rvm/gems/ruby-3.2.2/bin
     - /home/mww/.rvm/gems/ruby-3.2.2@global/bin
     - /home/mww/.rvm/rubies/ruby-3.2.2/bin
     - /home/mww/.rvm/bin
     - /sbin
     - /bin
     - /usr/sbin
     - /usr/bin
     - /usr/local/sbin
     - /usr/local/bin
     - /home/mww/bin

bundle env:

## Environment

```
Bundler             2.4.10
  Platforms         ruby, x86-freebsd-14
Ruby                3.2.2p53 (2023-03-30 revision e51014f9c05aa65cbf203442d37fef7c12390015) [x86-freebsd-14]
  Full Path         /home/mww/.rvm/rubies/ruby-3.2.2/bin/ruby
  Config Dir        /home/mww/.rvm/rubies/ruby-3.2.2/etc
RubyGems            3.4.10
  Gem Home          /home/mww/.rvm/gems/ruby-3.2.2
  Gem Path          /home/mww/.rvm/gems/ruby-3.2.2:/home/mww/.rvm/rubies/ruby-3.2.2/lib/ruby/gems/3.2.0
  User Home         /home/mww
  User Path         /home/mww/.local/share/gem/ruby/3.2.0
  Bin Dir           /home/mww/.rvm/gems/ruby-3.2.2/bin
Tools               
  Git               2.42.0
  RVM               1.29.12 (latest)
  rbenv             not installed
  chruby            If you use the C shell, add the following line to the .cshrc file in your
home directory to prevent core files from being written to disk:

	limit coredumpsize 0
		-- Dru <genesis@istar.ca>

  rubygems-bundler  (1.4.5)
```

## Bundler Build Metadata

```
Built At          2023-12-03
Git SHA           unknown
Released Version  false
```

### Gemfile

```ruby
source "https://rubygems.org"

ruby "3.2.2"

# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
gem "rails", "~> 7.1.2"

# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
gem "sprockets-rails"

# Use postgresql as the database for Active Record
gem "pg", "~> 1.1"

# Use the Puma web server [https://github.com/puma/puma]
gem "puma", ">= 5.0"

# Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
gem "importmap-rails"

# Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
gem "turbo-rails"

# Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
gem "stimulus-rails"

# Use Tailwind CSS [https://github.com/rails/tailwindcss-rails]
gem "tailwindcss-rails"

# Build JSON APIs with ease [https://github.com/rails/jbuilder]
gem "jbuilder"

# Use Redis adapter to run Action Cable in production
# gem "redis", ">= 4.0.1"

# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
# gem "kredis"

# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
# gem "bcrypt", "~> 3.1.7"

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem "tzinfo-data", platforms: %i[ windows jruby ]

# Reduces boot times through caching; required in config/boot.rb
gem "bootsnap", require: false

# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
# gem "image_processing", "~> 1.2"

group :development, :test do
  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
  gem "debug", platforms: %i[ mri windows ]
end

group :development do
  # Use console on exceptions pages [https://github.com/rails/web-console]
  gem "web-console"

  # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
  # gem "rack-mini-profiler"

  # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
  # gem "spring"
end

group :test do
  # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
  gem "capybara"
  gem "selenium-webdriver"
end
```

### Gemfile.lock

```
GEM
  remote: https://rubygems.org/
  specs:
    actioncable (7.1.2)
      actionpack (= 7.1.2)
      activesupport (= 7.1.2)
      nio4r (~> 2.0)
      websocket-driver (>= 0.6.1)
      zeitwerk (~> 2.6)
    actionmailbox (7.1.2)
      actionpack (= 7.1.2)
      activejob (= 7.1.2)
      activerecord (= 7.1.2)
      activestorage (= 7.1.2)
      activesupport (= 7.1.2)
      mail (>= 2.7.1)
      net-imap
      net-pop
      net-smtp
    actionmailer (7.1.2)
      actionpack (= 7.1.2)
      actionview (= 7.1.2)
      activejob (= 7.1.2)
      activesupport (= 7.1.2)
      mail (~> 2.5, >= 2.5.4)
      net-imap
      net-pop
      net-smtp
      rails-dom-testing (~> 2.2)
    actionpack (7.1.2)
      actionview (= 7.1.2)
      activesupport (= 7.1.2)
      nokogiri (>= 1.8.5)
      racc
      rack (>= 2.2.4)
      rack-session (>= 1.0.1)
      rack-test (>= 0.6.3)
      rails-dom-testing (~> 2.2)
      rails-html-sanitizer (~> 1.6)
    actiontext (7.1.2)
      actionpack (= 7.1.2)
      activerecord (= 7.1.2)
      activestorage (= 7.1.2)
      activesupport (= 7.1.2)
      globalid (>= 0.6.0)
      nokogiri (>= 1.8.5)
    actionview (7.1.2)
      activesupport (= 7.1.2)
      builder (~> 3.1)
      erubi (~> 1.11)
      rails-dom-testing (~> 2.2)
      rails-html-sanitizer (~> 1.6)
    activejob (7.1.2)
      activesupport (= 7.1.2)
      globalid (>= 0.3.6)
    activemodel (7.1.2)
      activesupport (= 7.1.2)
    activerecord (7.1.2)
      activemodel (= 7.1.2)
      activesupport (= 7.1.2)
      timeout (>= 0.4.0)
    activestorage (7.1.2)
      actionpack (= 7.1.2)
      activejob (= 7.1.2)
      activerecord (= 7.1.2)
      activesupport (= 7.1.2)
      marcel (~> 1.0)
    activesupport (7.1.2)
      base64
      bigdecimal
      concurrent-ruby (~> 1.0, >= 1.0.2)
      connection_pool (>= 2.2.5)
      drb
      i18n (>= 1.6, < 2)
      minitest (>= 5.1)
      mutex_m
      tzinfo (~> 2.0)
    addressable (2.8.5)
      public_suffix (>= 2.0.2, < 6.0)
    base64 (0.2.0)
    bigdecimal (3.1.4)
    bindex (0.8.1)
    bootsnap (1.17.0)
      msgpack (~> 1.2)
    builder (3.2.4)
    capybara (3.39.2)
      addressable
      matrix
      mini_mime (>= 0.1.3)
      nokogiri (~> 1.8)
      rack (>= 1.6.0)
      rack-test (>= 0.6.3)
      regexp_parser (>= 1.5, < 3.0)
      xpath (~> 3.2)
    concurrent-ruby (1.2.2)
    connection_pool (2.4.1)
    crass (1.0.6)
    date (3.3.4)
    debug (1.8.0)
      irb (>= 1.5.0)
      reline (>= 0.3.1)
    drb (2.2.0)
      ruby2_keywords
    erubi (1.12.0)
    globalid (1.2.1)
      activesupport (>= 6.1)
    i18n (1.14.1)
      concurrent-ruby (~> 1.0)
    importmap-rails (1.2.3)
      actionpack (>= 6.0.0)
      activesupport (>= 6.0.0)
      railties (>= 6.0.0)
    io-console (0.6.0)
    irb (1.9.1)
      rdoc
      reline (>= 0.3.8)
    jbuilder (2.11.5)
      actionview (>= 5.0.0)
      activesupport (>= 5.0.0)
    loofah (2.22.0)
      crass (~> 1.0.2)
      nokogiri (>= 1.12.0)
    mail (2.8.1)
      mini_mime (>= 0.1.1)
      net-imap
      net-pop
      net-smtp
    marcel (1.0.2)
    matrix (0.4.2)
    mini_mime (1.1.5)
    mini_portile2 (2.8.5)
    minitest (5.20.0)
    msgpack (1.7.2)
    mutex_m (0.2.0)
    net-imap (0.4.7)
      date
      net-protocol
    net-pop (0.1.2)
      net-protocol
    net-protocol (0.2.2)
      timeout
    net-smtp (0.4.0)
      net-protocol
    nio4r (2.7.0)
    nokogiri (1.15.5)
      mini_portile2 (~> 2.8.2)
      racc (~> 1.4)
    nokogiri (1.15.5-x86_64-linux)
      racc (~> 1.4)
    pg (1.5.4)
    psych (5.1.1.1)
      stringio
    public_suffix (5.0.4)
    puma (6.4.0)
      nio4r (~> 2.0)
    racc (1.7.3)
    rack (3.0.8)
    rack-session (2.0.0)
      rack (>= 3.0.0)
    rack-test (2.1.0)
      rack (>= 1.3)
    rackup (2.1.0)
      rack (>= 3)
      webrick (~> 1.8)
    rails (7.1.2)
      actioncable (= 7.1.2)
      actionmailbox (= 7.1.2)
      actionmailer (= 7.1.2)
      actionpack (= 7.1.2)
      actiontext (= 7.1.2)
      actionview (= 7.1.2)
      activejob (= 7.1.2)
      activemodel (= 7.1.2)
      activerecord (= 7.1.2)
      activestorage (= 7.1.2)
      activesupport (= 7.1.2)
      bundler (>= 1.15.0)
      railties (= 7.1.2)
    rails-dom-testing (2.2.0)
      activesupport (>= 5.0.0)
      minitest
      nokogiri (>= 1.6)
    rails-html-sanitizer (1.6.0)
      loofah (~> 2.21)
      nokogiri (~> 1.14)
    railties (7.1.2)
      actionpack (= 7.1.2)
      activesupport (= 7.1.2)
      irb
      rackup (>= 1.0.0)
      rake (>= 12.2)
      thor (~> 1.0, >= 1.2.2)
      zeitwerk (~> 2.6)
    rake (13.1.0)
    rdoc (6.6.0)
      psych (>= 4.0.0)
    regexp_parser (2.8.2)
    reline (0.4.1)
      io-console (~> 0.5)
    rexml (3.2.6)
    ruby2_keywords (0.0.5)
    rubyzip (2.3.2)
    selenium-webdriver (4.15.0)
      rexml (~> 3.2, >= 3.2.5)
      rubyzip (>= 1.2.2, < 3.0)
      websocket (~> 1.0)
    sprockets (4.2.1)
      concurrent-ruby (~> 1.0)
      rack (>= 2.2.4, < 4)
    sprockets-rails (3.4.2)
      actionpack (>= 5.2)
      activesupport (>= 5.2)
      sprockets (>= 3.0.0)
    stimulus-rails (1.3.0)
      railties (>= 6.0.0)
    stringio (3.1.0)
    tailwindcss-rails (2.0.32)
      railties (>= 6.0.0)
    tailwindcss-rails (2.0.32-x86_64-linux)
      railties (>= 6.0.0)
    thor (1.3.0)
    timeout (0.4.1)
    turbo-rails (1.5.0)
      actionpack (>= 6.0.0)
      activejob (>= 6.0.0)
      railties (>= 6.0.0)
    tzinfo (2.0.6)
      concurrent-ruby (~> 1.0)
    web-console (4.2.1)
      actionview (>= 6.0.0)
      activemodel (>= 6.0.0)
      bindex (>= 0.4.0)
      railties (>= 6.0.0)
    webrick (1.8.1)
    websocket (1.2.10)
    websocket-driver (0.7.6)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.5)
    xpath (3.2.0)
      nokogiri (~> 1.8)
    zeitwerk (2.6.12)

PLATFORMS
  x86-freebsd-14
  x86_64-linux

DEPENDENCIES
  bootsnap
  capybara
  debug
  importmap-rails
  jbuilder
  pg (~> 1.1)
  puma (>= 5.0)
  rails (~> 7.1.2)
  selenium-webdriver
  sprockets-rails
  stimulus-rails
  tailwindcss-rails
  turbo-rails
  tzinfo-data
  web-console

RUBY VERSION
   ruby 3.2.2p53

BUNDLED WITH
   2.4.10

Thanks for your attention.

**Comment by whitehome** [comment]
Noticed this in the bundler ENV output:

tailwindcss-rails (2.0.32-x86_64-linux)

Are you checking linux vs unix?  Does it really matter or just a test failure?

**Comment by flavorjones** [comment]
> Is this intentional or an error that FreeBSD is not supported by TailwindCSS-rails. If so is there a work around?

This gem, `tailwindcss-rails`, repackages the upstream executable binaries from https://github.com/tailwindlabs/tailwindcss/releases. That project does not provide freebsd executables. If you'd like them to support your platform, please make a request upstream at that project.

Alternatively, you may install `tailwindcss` yourself, and follow the instructions at https://github.com/rails/tailwindcss-rails#using-a-local-installation-of-tailwindcss to have this gem use those executables.

**Reply by whitehome** [reply]
Ok thank you for confirming what I suspected.
